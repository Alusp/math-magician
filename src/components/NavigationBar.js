import React from 'react';
import {BrowserRouter as Router,   Link } from 'react-router-dom';

function NavigationBar() {
  return (
    <Router>
    <div className="py-6 px-16 shadow-lg flex justify-between">
      <Link to="/" className="text-3xl font-semibold text-orange-500">
        Math Magician
      </Link>
      <div className="flex">
        <Link to="/" exact className="mx-6 hover:text-orange-500 text-xl">
          Home
        </Link>
        <Link to="/calculator" exact className="mx-6 hover:text-orange-500 text-xl">
          Calculator
        </Link>
        <Link to="/quote" exact className="ml-6 hover:text-orange-500 text-xl">
          Quotes
        </Link>
      </div>
    </div>
    </Router>
  );
}

export default NavigationBar;
