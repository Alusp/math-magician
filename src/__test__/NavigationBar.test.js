import React from 'react';
import renderer from 'react-test-renderer';
import NavigationBar from '../components/NavigationBar';
import '@testing-library/jest-dom';

describe('Test NavigationBar', () => {
  test('Testing NavigationBar', () => {
    const tree = renderer.create(<NavigationBar />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
