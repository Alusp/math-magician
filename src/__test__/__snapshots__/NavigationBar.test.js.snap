// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test NavigationBar Testing NavigationBar 1`] = `
<div
  className="py-6 px-16 shadow-lg flex justify-between"
>
  <a
    className="text-3xl font-semibold text-orange-500"
    href="/"
    onClick={[Function]}
  >
    Math Magician
  </a>
  <div
    className="flex"
  >
    <a
      className="mx-6 hover:text-orange-500 text-xl"
      exact={true}
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      className="mx-6 hover:text-orange-500 text-xl"
      exact={true}
      href="/calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <a
      className="ml-6 hover:text-orange-500 text-xl"
      exact={true}
      href="/quote"
      onClick={[Function]}
    >
      Quotes
    </a>
  </div>
</div>
`;
